examples/lxmert/requirements.txt

```
@@ -0,0 +1,105 @@
+absl-py==0.9.0
+apipkg==1.5
+appdirs==1.4.3
+astunparse==1.6.3
+attrs==19.3.0
+black==20.8b1
+CacheControl==0.12.6
+cachetools==4.1.1
+cairocffi==1.1.0
+certifi==2020.6.20
+cffi==1.14.2
+chardet==3.0.4
+click==7.1.2
+colorama==0.4.3
+contextlib2==0.6.0
+cycler==0.10.0
+distlib==0.3.0
+distro==1.4.0
+execnet==1.7.1
+filelock==3.0.12
+flake8==3.8.3
+fugashi==1.0.4
+future==0.18.2
+gast==0.3.3
+google-auth==1.19.0
+google-auth-oauthlib==0.4.1
+google-pasta==0.2.0
+grpcio==1.30.0
+h5py==2.10.0
+html5lib==1.0.1
+idna==2.8
+ipaddr==2.2.0
+ipadic==1.0.0
+isort==5.4.2
+joblib==0.16.0
+Keras-Preprocessing==1.1.2
+kiwisolver==1.2.0
+lockfile==0.12.2
+Markdown==3.2.2
+matplotlib==3.3.1
+mccabe==0.6.1
+mecab-python3==0.996.5
+more-itertools==8.4.0
+msgpack==0.6.2
+mypy-extensions==0.4.3
+numpy==1.19.0
+oauthlib==3.1.0
+opencv-python==4.4.0.42
+opt-einsum==3.2.1
+packaging==20.3
+pathspec==0.8.0
+pep517==0.8.2
+Pillow==7.2.0
+plac==1.2.0
+pluggy==0.13.1
+progress==1.5
+protobuf==3.12.2
+psutil==5.7.0
+py==1.9.0
+pyaml==20.4.0
+pyasn1==0.4.8
+pyasn1-modules==0.2.8
+pycodestyle==2.6.0
+pycparser==2.20
+pyflakes==2.2.0
+pyparsing==2.4.6
+pytest==5.4.3
+pytest-forked==1.2.0
+pytest-xdist==1.33.0
+python-dateutil==2.8.1
+pytoml==0.1.21
+PyYAML==5.3.1
+regex==2020.6.8
+requests==2.22.0
+requests-oauthlib==1.3.0
+retrying==1.3.3
+rsa==4.6
+sacremoses==0.0.43
+scikit-learn==0.23.1
+scipy==1.5.1
+sentencepiece==0.1.91
+six==1.14.0
+tensorboard==2.2.2
+tensorboard-plugin-wit==1.7.0
+tensorflow==2.2.0
+tensorflow-estimator==2.2.0
+termcolor==1.1.0
+threadpoolctl==2.1.0
+timeout-decorator==0.4.1
+tokenizers==0.8.1rc2
+toml==0.10.1
+torch==1.6.0
+torchvision==0.7.0
+tqdm==4.47.0
+-e git+https://github.com/eltoto1219/transformers.git@4ed21b4e2fd0c2b0acb8b40c0cb43bbf247b44ab#egg=transformers
```

Is that needed? Are you installing from your fork?
```suggestion
transformers==3.1.0
```